tests/jobs/rft/test_gendata_rft.py

```diff
@@ -425,3 +426,222 @@ def test_defaults():
     # And compare with argparse:
     assert csv_job_default == _build_parser().get_default("csvfile")
     assert directory_job_default == _build_parser().get_default("outputdirectory")
+
+
+@pytest.mark.ert_integration
+@pytest.mark.skipif(
+    sys.platform == "darwin",
+    reason=(
+        "Forward models have same name as executable and "
+        "libres will find wrong exe on case insensitive system "
+        "bug report here: https://github.com/equinor/libres/issues/1053"
+    ),
```

Comment:
Can we not remove this then? As we have fixed the issue of case sensitivity

Reply:
Done.